model/site.go

```diff
@@ -40,7 +40,11 @@ const typeSite = "Site"
 type Site struct {
 	Skey         int64
 	Name         string
+	Description  string
```

Comment:
Doesn't have to happen in this PR but i think these fields should have comments seeings as they're exported.